### PR TITLE
Improved user menu on mobile

### PR DESCRIPTION
**Pull request by mllocs**
http://g.recordit.co/aMnJR1VuBq.gif

Yet another lifting in our top navigation bar. Now I'm removing the unnecessary dropdown inception.

### CODE REVIEW

File: app/views/application/_navbar.html.erb [hunk 1]
```diff
@@ -19,16 +19,23 @@
         </div>
         <div class="col-xs-12 col-sm-9 col-md-9 col-lg-9">
           <div id="navbar-collapse" class="collapse navbar-collapse">
-            <ul class="nav navbar-nav navbar-right">
+            <ul class="nav navbar-nav navbar-right hidden-xs">
               <% if current_user %>
-                <%= render 'application/menus/user_admin_menu' %>
+                  <%= render 'application/menus/user_admin_menu' %>
```
Comment (markets) [inline app/views/application/_navbar.html.erb]: :eyes: indentation

File: app/views/application/_navbar.html.erb [hunk 2]
```diff
@@ -19,16 +19,23 @@
...
+              <% end %>
+            </ul>
+
+            <ul class="nav navbar-nav navbar-right hidden-sm hidden-md hidden-lg">
+              <% if current_user %>
+                  <li>
+                    <%= image_tag avatar_url(current_user) %>
+                    <%= current_user.email %>
+                  </li>
+                  <%= render 'application/menus/user_admin_menu_items' %>
```
Comment (markets) [inline app/views/application/_navbar.html.erb]: :eyes: indentation of this block

File: app/views/application/menus/_user_admin_menu_items.html.erb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<li class="divider" role="presentation"></li>
+
+<li role="presentation">
+  <%= link_to destroy_user_session_path, method: :delete do %>
+    <%= glyph :log_out %>
+    <%= t "application.navbar.sign_out" %>
+  <% end %>
+</li>
+
+
```
Comment (markets) [inline app/views/application/menus/_user_admin_menu_items.html.erb]: :scissors: extra empty line?

**Comment by enricostano** [comment]
@mllocs `feature/move-language-selector-to-footer` branch has been merged I'm pointing this PR back to `develop` and deploy to staging

**Comment by enricostano** [comment]
Deployed to https://staging.timeoverflow.org

@mllocs please give a look and :shipit:

**Comment by enricostano** [comment]
Deployed again to https://staging.timeoverflow.org/

**Comment by enricostano** [comment]
@mllocs say it's OK :green_apple:

**Pull request merged**